### PR TITLE
Update helmich/phpunit-json-assert

### PR DESCRIPTION
**Pull request by martinssipenko**

### CODE REVIEW

**Comment by martinssipenko** [comment]
@martin-helmich dependabot has not kicked in yet so I did a manual update. Could I ask you to take a look at this and do a release after merge?

**Comment by martin-helmich** [comment]
:+1: Thanks for the PR.

On a side note, I found out why dependabot decided to remain inactive -- it seems like depedabot is much stricter regarding updates than Composer itself. Apparently, this libraries PHP requirement is `^7.1`, whereas `helmich/phpunit-json-assert` in `3.4` has `^7.2`.

De facto, this update raises the PHP requirement of this library from `^7.1` to `^7.2` -- which I can live with, since 7.1 has been EOL for over a year anyway (as is 7.2, as of recently).

**Comment by martinssipenko** [comment]
@martin-helmich shall I make a required PHP version but too?

File: composer.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
     ],
     "require": {
         "php": "^7.1 || ^8.0",
```
Comment (martin-helmich) [inline composer.json:12]: As discussed earlier.

```suggestion
        "php": "^7.2 || ^8.0",
```

**Comment by martin-helmich** [comment]
>shall I make a required PHP version bump too?

Just had the same thought. Added a suggestion. :)

**Review by martin-helmich — APPROVED** [review]

**Comment by martin-helmich** [comment]
:+1: Merged. Release is [also published](https://github.com/martin-helmich/phpunit-psr7-assert/releases/tag/v4.3.0).

**Comment by martinssipenko** [comment]
> 👍 Merged. Release is [also published](https://github.com/martin-helmich/phpunit-psr7-assert/releases/tag/v4.3.0).

Thank you very much!

**Pull request merged**